### PR TITLE
Support a more complex sass library

### PR DESCRIPTION
**Pull request by flyingmachine**
This change was motivated by my need to support a Bower installation of the Susy sass library.

Changes:
- Usig jsass `5.2.0` instead of `3.3.1`. The API changed quite a bit. One nice thing about `5.2.0` is that `Import` objects have a `.getAbsoluteUri` method that returns an Import's directory
- I removed the following from `core.clj`:

``` clojure
(find-resource (io/resource (add-underscore url)))      
(find-resource (io/resource url))
```

It looks like these were used to support the use case in the test file where we were creating a file in a temporary directory, and trying to import resource files from the created file. I couldn't see why one might want this functionality.

Still, that's not enough reason to remove the functionality. The larger reason is that those lines were causing the `susy` import to throw the exception `An import loop has been found`.

### CODE REVIEW

**Comment by Deraen** [comment]
Thanks!

I'l try to remember why those lines existed and if removing those has some other consequences.

Other thing, I don't want 4000 lines of external code added to the repo. Either:
- Import Susy from WebJars: `[org.webjars.bower/susy "2.2.12"]` and `@import "susy/sass/susy";` or
- Write small local test case

File: build.boot [hunk 1]
```diff
@@ -1,19 +1,19 @@
-(def +version+ "0.2.1")
+(def +version+ "0.2.2")
 
 (set-env!
-  :resource-paths #{"src" "boot-sass/src" "lein-sass4clj/src"}
```
Comment (Deraen) [inline build.boot:4]: Please revert whitespace / indent changes.

File: test/sass4clj/core_test.clj [hunk 2]
```diff
@@ -3,29 +3,8 @@
             [sass4clj.core :refer :all])
   (:import [java.io File]))
 
-(def sass
```
Comment (Deraen) [inline test/sass4clj/core_test.clj:6]: What is the reason to remove these tests?

Comment (Deraen) [inline test/sass4clj/core_test.clj:6]: Hmm, read the message again. I'm quite sure these tests represent real use case needed by lein/boot tests.

Comment (flyingmachine) [inline test/sass4clj/core_test.clj:6]: The tests were passing, but I didn't understand what use case they were meant to represent, and they relied on the code I removed from `core.clj`.

It also seemed like they created ambiguity that runs counter to how Sass is used. `@import` is supposed to take a relative path that resolves to a single absolute path, but these tests were treating the import paths as relative to both the temporary file and the resources directory.

I'm sure I'm not understanding it correctly, but included the change in order to get your thoughts. I couldn't think of a case where you'd want a a sass file in a completely random location to be able to `@import` files relative to resources, and not just `@import` relative to its own location, which seems to be the standard behavior 😄 

Thanks for having a look! This library is very useful and well done.

Comment (Deraen) [inline test/sass4clj/core_test.clj:6]: > It also seemed like they created ambiguity that runs counter to how Sass is used. @import is supposed to take a relative path that resolves to a single absolute path, but these tests were treating the import paths as relative to both the temporary file and the resources directory.

I copied the logic used by the less4clj so it is possible that it doesn't completely make sense here.

**Comment by Deraen** [comment]
I updated Jsass in another commit: https://github.com/Deraen/sass4clj/commit/41ac016018174c3c2848bc694da44f8ee4bfe0b4

**Pull request closed without merging**